PWGCF/FEMTOSCOPY/macros/AddTaskFemtoDreamDeuteron.C

```diff
@@ -55,30 +55,64 @@ AliAnalysisTaskSE* AddTaskFemtoDreamDeuteron(
   TrackCuts1->SetCutSharedCls(true);
   TrackCuts1->SetCutTPCCrossedRows(true,70,0.83);
   //Here you define the PID
-  TrackCuts1->SetPID(AliPID::kDeuteron, 0.8);
+  TrackCuts1->SetPID(AliPID::kDeuteron, 1.4);
   //We are looking for pions rejecting them would be obstructive.
   TrackCuts1->SetRejLowPtPionsTOF(true);
   //this checks if the sigma of the wanted hypothesis is the smallest, and if
   //another particle has a smaller sigma, the track is rejected.
   TrackCuts1->SetCutSmallestSig(true);
 
-  //The same things for negative pions
+  //The same things for antideuterons
   AliFemtoDreamTrackCuts *TrackCuts2=new AliFemtoDreamTrackCuts();
   TrackCuts2->SetIsMonteCarlo(isMC);
   TrackCuts2->SetCutCharge(-1);
   TrackCuts2->SetFilterBit(128);
-  TrackCuts2->SetPtRange(0.4, 3.0);
+  TrackCuts2->SetPtRange(0.4, 4.0);
   TrackCuts2->SetEtaRange(-0.8, 0.8);
   TrackCuts2->SetNClsTPC(80);
   TrackCuts2->SetDCAReCalculation(true);//Get the dca from the PropagateToVetex
   TrackCuts2->SetDCAVtxZ(0.2);
   TrackCuts2->SetDCAVtxXY(0.1);
   TrackCuts2->SetCutSharedCls(true);
   TrackCuts2->SetCutTPCCrossedRows(true,70,0.83);
-  TrackCuts2->SetPID(AliPID::kDeuteron, 0.8);
+  TrackCuts2->SetPID(AliPID::kDeuteron, 1.4);
   TrackCuts2->SetRejLowPtPionsTOF(true);
   TrackCuts2->SetCutSmallestSig(true);
 
+  //protons
+  AliFemtoDreamTrackCuts *TrackCuts3=new AliFemtoDreamTrackCuts();
```

Comment:
consider to use the static function of the track cuts to properly set the proton cuts
  AliFemtoDreamTrackCuts *TrackCuts =
      AliFemtoDreamTrackCuts::PrimProtonCuts(isMC, true, false, false);
  TrackCuts->SetCutCharge(1);

  AliFemtoDreamTrackCuts *AntiTrackCuts =
      AliFemtoDreamTrackCuts::PrimProtonCuts(isMC, true, false, false);
  AntiTrackCuts->SetCutCharge(-1);